skcosmo/selection/FPS.py

```diff
@@ -0,0 +1,168 @@
+from abc import abstractmethod
+import numpy as np
+import scipy
+
+from ._base import _BaseSelection
+
+def calculate_pcov_distances(points, ref_idx, idxs=None):
```

Comment:
this needs a docstring, and looks like a private function, so it should be named `_calculate_pcov_distances`